app/experimenter/static/rapid/components/forms/ExperimentForm.tsx

```diff
@@ -198,10 +198,14 @@ export const SettingsForm: React.FC = () => {
 
 export const BranchesForm: React.FC = () => {
   const formData = useExperimentState();
+
   const variants =
     [...formData.variants].sort((a, b) =>
       a.is_control < b.is_control ? 1 : -1,
     ) || [];
+  const [variantRatio, setVariantRatio] = React.useState<string>(
```

Comment:
You don't need to use any state here. Just create a function that you can pass to the `value` prop of the input. Something like:
```tsx
  function ratioToPercentage(ratio: number) {
    const ratioTotal: number = variants.reduce((a, b) => a + b.ratio, 0);
    return Math.round((ratio / ratioTotal) * 10000) / 100;
  }
```

Also, I'd probably go with `Math.round` here since it will only display decimals *if necessary*
Which you can then call later:
```tsx
<input value={ratioToPercentage(variant.ratio)} />
```